test-it/ViaTests/PolymorphicLoopBug.via

```diff
@@ -0,0 +1,119 @@
+// Autogenerated Vireo assembly file
```

Comment:
There's nothing about the bug fix that has any potential to be platform-specific, so it should, but I did not test it.   I'm not going to do extra work to build and test it manually when we're already working on a separate Windows CI.